pkg/web_app/lib/src/search.dart

```diff
@@ -15,7 +15,8 @@ void setupSearch() {
 }
 
 final _acceptedSearchPaths = <Pattern>[
-  '/packages',
+  RegExp(r'^/packages$'),
+  RegExp(r'^/packages\?'),
   '/dart/packages',
   '/flutter/packages',
```

Comment:
```dart
RegExp(r'^/(:?dart/|flutter/)?packages(?:\?|$)')
```
Or
```dart
RegExp(r'^/packages(?:\?|$)')
RegExp(r'^/dart/packages(?:\?|$)')
RegExp(r'^/flutter/packages(?:\?|$)')
```

So we're being as narrow as reasonably possible with the `/dart/packages` and `/flutter/packages` urls too.. I assume these have to be the first part of the thing..

Reply:
nit: also `https://pub.dev/flutter/favorites` is a list page.